ldk/go/examples/search-searchbar/Makefile

```diff
@@ -0,0 +1,54 @@
+REPO := github.com/open-olive/loop-development-kit/ldk/go/example/clipboard
```

Comment:
Should be:
` REPO := github.com/open-olive/loop-development-kit/ldk/go/example/search-searchbar`